modules/ROOT/pages/datagraph-cli-errors.adoc

```diff
@@ -0,0 +1,227 @@
+= DataGraph CLI Errors
+
+When using the Datagraph CLI, you might encounter errors. The errors are divided in two categories:
+
+. Command errors: These are errors you can encounter when using incorrect arguments or options in commands.
+. Validation errors: These are errors you can encounter with validation when adding new API sources or when updating sources in the unified schema. 
```

Comment:
These are errors you can encounter with validation when > These errors occur during validation when...